src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/RespondToDefenceSpecCallbackHandler.java

```diff
@@ -129,6 +129,7 @@ protected Map<String, Callback> callbacks() {
             .put(callbackKey(V_1, MID, "set-mediation-show-tag"), this::setMediationShowTag)
             .put(callbackKey(ABOUT_TO_SUBMIT), params -> aboutToSubmit(params, false))
             .put(callbackKey(V_1, ABOUT_TO_SUBMIT), params -> aboutToSubmit(params, true))
+            .put(callbackKey(V_2, ABOUT_TO_SUBMIT), params -> aboutToSubmit(params, false))
```

Comment:
just checking, but why do you want to stop doing the changes from v1, including initialization of the court location drop down?

Reply:
Yes, agree, I had that doubt, but now it's clear, so I'll change it. Thanks!